tests/e2e/monitoring/test_monitoring_after_reboot_master_node.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import tier4, E2ETest, ignore_leftovers
+from ocs_ci.utility import aws
+from tests.sanity_helpers import Sanity
+from ocs_ci.ocs.monitoring import check_pvcdata_collected_on_prometheus
+from ocs_ci.ocs.node import wait_for_nodes_status, get_typed_nodes
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed, TimeoutExpiredError
+
+
+logger = logging.getLogger(__name__)
+
+
+@retry((CommandFailed, TimeoutExpiredError), tries=10, delay=3, backoff=1)
+def wait_for_master_node_to_be_running_state():
```

Comment:
I think this function is not needed. Instead, you can call directly to wait_for_nodes_status() and in case needed, pass a 
 greater timeout

Reply:
When master is down will get this below error, for sometime couldn't run any oc commands. So added retry iterator.
```
E           ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc --kubeconfig /home/akarsha/aws-install/sep-09/auth/kubeconfig get Node ip-10-0-168-220.us-east-2.compute.internal -o yaml.
E           Error is error: You must be logged in to the server (Unauthorized)
```
Agreed on TimeoutExpiredError, will pass the greater timeout.